contracts/scripts/compileSol.sh

```diff
@@ -8,16 +8,18 @@ echo 'Building contracts'
 # Delete old files
 rm -rf ./compiled/*
 
-mkdir -p ./compiled/abis
-
-if [[ $* == *--native* ]]; then
-    # You need Solidity 0.5.16 (https://github.com/ethereum/solidity/releases/tag/v0.5.16) installed in your PATH
-    SOLC_VERSION=native
+if [[ -z "${SOLC_PATH}" ]]; then
+    # Assumes that you have solc 0.5.17 (https://github.com/ethereum/solidity/releases/tag/v0.5.17) installed in your PATH
+    solcBin="solc"
 else
-    SOLC_VERSION=0.5.16
+    # Otherwise, you can specify the path to solc 0.5.17
+    solcBin="${SOLC_PATH}"
 fi
```

Comment:
This will require developer to install solc manually. Can we automate this by downloading the solc binary like we do in CI environment?

Another option would be using solc-js but it requires additional work because solc-js has different interface (I tried it yesterday but didn't make much progress).

Reply:
> Can we automate this by downloading the solc binary like we do in CI environment?

Great idea! Will do.